packages/flutter_tools/test/application_package_test.dart

```diff
@@ -18,13 +18,21 @@ import 'src/context.dart';
 
 void main() {
   group('ApkManifestData', () {
-    testUsingContext('parse sdk', () {
-      final ApkManifestData data =
-          ApkManifestData.parseFromAaptBadging(_aaptData);
+    testUsingContext('parse sdk with explicit enabled activity', () {
```

Comment:
[nit] Do we need the context in these tests?
[nit] I'd name these tests from the intended functionality of `ApkManifestData`, as in `selects explicitly enabled main activity`.